### DISCUSSION TITLE
FR: Add a way to scroll horizontally between audio devices in mixer window

### DISCUSSION THREAD

**Discussion by MBoutray** [File-New-Project/EarTrumpet#632] [Ideas]
Hello there ! I just add a small idea to improve this amazing app.
I think it would be useful to add a key combination to scroll horizontally in the volume mixer window, something like ctrl + scroll wheel or shift + scroll wheel. I believe it would more ergonomic than having to drag the bottom bar, just a minor improvement imho.
I'm not really used to making feature requests as I am a beginner on Github, please ask me to clarify if my idea wasn't really clear.

**Comment by vertigo220** [comment]
This wouldn't even need a modifier key. It could (should) just scroll whenever the user scrolls in empty space at the bottom of an audio device "column" or in the space where the audio device names are at the top.

I'd also like to add that one of the issues I was going to post was that the mixer window wasn't showing all of my devices, but as it turns out, it does, I just didn't even realize that I had to scroll to see more than three. With the Windows setting to auto-collapse scroll bars enabled (and possibly made worse by dark theme, as that's what I use, so I don't know if it's better in light theme), the scroll bar is extremely easy to completely overlook.
